### PR TITLE
MCSC-1016 Move BCE page and add kickout status

### PR DESCRIPTION
**Pull request by mrah93**

### CODE REVIEW

File: app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+package pages.setupquestions.lifetimeallowance
+
+import models.KickOutStatus
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object KickOutStatusPage extends QuestionPage[KickOutStatus] {
+
+  override def path: JsPath = JsPath \ "setup" \ "lta" \ toString
```
Comment (MichaelAAbbott) [inline app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala:25]: should this be lta?

File: app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ * limitations under the License.
+ */
+
+package pages.setupquestions.lifetimeallowance
+
+import models.KickOutStatus
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object KickOutStatusPage extends QuestionPage[KickOutStatus] {
```
Comment (MichaelAAbbott) [inline app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala:23]: should this be a "page"?

Comment (MichaelAAbbott) [reply app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala:23]: or should this just all be part of it's own model?

File: test/pages/setupquestions/lifetimeallowance/KickOutStatusPageSpec.scala [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package pages.setupquestions.lifetimeallowance
+
+import models.KickOutStatus
+import org.scalacheck.{Arbitrary, Gen}
+import pages.behaviours.PageBehaviours
+class KickOutStatusPageSpec extends PageBehaviours {
+
+  "KickOutStatusPage" - {
```
Comment (MichaelAAbbott) [inline test/pages/setupquestions/lifetimeallowance/KickOutStatusPageSpec.scala:8]: Again, should this be a page?

File: app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala (hunk 1 above)
Comment (MichaelAAbbott) [reply app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala:25]: as this kickoutstatus stores data for lta and aa right?

Comment (mrah93) [reply app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala]: moved to kickoutStatus on top level

File: app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala (hunk 2 above)
Comment (mrah93) [reply app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala]: moved to model

File: test/pages/setupquestions/lifetimeallowance/KickOutStatusPageSpec.scala (hunk 3 above)
Comment (mrah93) [reply test/pages/setupquestions/lifetimeallowance/KickOutStatusPageSpec.scala]: removed and moved to model

**Review by MichaelAAbbott — APPROVED** [review]

**Pull request merged**